### ISSUE TITLE
Arabic Finetuning

### ISSUE THREAD

**Issue by MohieEldinMuhammad** [PaddlePaddle/PaddleOCR#7548]
![image](https://user-images.githubusercontent.com/89604782/189310610-eb2dc0c1-03e8-4599-9cba-40acada81322.png)
تتمكن
ِقطع كَلُ َعَة َّي
أن
هلا
تجمع
على
كان
T .
تستطع
ولكنها
لهم
خمإرا
بها
من
تفرئى
أن
حاولى
لَها
فَقالّ
يساعدَها
أن
أبيها
كجإ
ك
ولكتنى
فقالت
فقطع
قطعة
```

Why the detection is not capturing line by line (like in English) ? may this is related to the training dataset so if i finetuned it with sentences it will get better ? 
also is training the recognizer depends in in way on the detection or it's just the image and ground truth text in the whole image ?

**Comment by MohieEldinMuhammad** [comment]
To illustrate more my second question, how the end result of the model map each detected box to the text inside it if both detection and recognition are done independently ?

**Comment by andyjpaddle** [comment]
you can try to increase the unclip_ratio, also finetune with sentence will get better. 
detection and recognition are finished step by step, so they can map using the bbox coordinate.

**Comment by MohieEldinMuhammad** [comment]
Thanks for your help,

Okay great i will fine-tune on sentence and paragraphs but should i include also words or sentences will be sufficient? 

Also can you illustrate more on the relationship between training the recognizer and detector?

**Comment by andyjpaddle** [comment]
it depends on what model are you going to train?
For detector, you should prepare your dataset according to this https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/doc/doc_en/dataset/ocr_datasets_en.md#11-paddleocr-text-detection-format-annotation , which aims to locate the text location
For recognizer, you should prepare your dataset according to this https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/doc/doc_en/dataset/ocr_datasets_en.md#21-paddleocr-text-recognition-format-annotation , which aims to recognize what the text line cropped from detection bbox is

**Comment by MohieEldinMuhammad** [comment]
So i should not collect whole images but Rather cropped images with the
text as if it was cropped with the detector coordinates
(That's in case i will make a recognition training independent to detection
training)
Another option is to give the whole image to detector and crop the image
based on the boxes i got and train the recognizer. Did i understand you
well ?

**Comment by andyjpaddle** [comment]
detection and recognition is two different task. For detection, you should prepare whole images which contained a few text lines and labels which are some bboxes represent the location of these text lines. For recognition, you should prepare text line image which only contained a line text and label which is a line word. I suggest you search this two task firstly and then start your experiments

**Comment by MohieEldinMuhammad** [comment]
Thanks a lot for your help
